server/aws-lsp-codewhisperer/src/language-server/qChatServer.ts

```diff
@@ -0,0 +1,39 @@
+import {
+    CancellationToken,
+    ChatParams,
+    CredentialsProvider,
+    EndChatParams,
+    Server,
+} from '@aws/language-server-runtimes/server-interface'
+import { ChatSessionManagementService } from './chat/chatSessionManagementService'
+
+export const QChatServerToken =
```

Comment:
nit: for the time being we can call this QChatServer given there is no possibility to use streaming client with iam credentials